addons/mail/models/mail_mail.py

```diff
@@ -3,12 +3,10 @@
 import base64
 import datetime
 import logging
+import psycopg2
```

Comment:
Not an stdlib import, should be in a separate group between stdlib imports and "local" (openerp) imports.

Also maybe switch openerp imports to odoo?